README.md

```diff
@@ -13,21 +13,29 @@ There are several ways to install this app onto a workload cluster.
 
 ## Upgrading cilium version
 
-The contents of the `helm` folder are being generated by the `make` target called `make update-chart`.
+The contents of the `helm` folder are being generated by the `make` target called `make update-chart`. You can use it like this
+
+```
+APPLICATION=cilium make update-chart
+```
 
 This target uses [`vendir`](https://carvel.dev/vendir/) to fetch the helm chart contained in [the fork of the cilium repository that we maintain](https://github.com/giantswarm/cilium-upstream).
 Currently, the `main` branch on the fork contains the upstream tag `v1.14.3`, with our custom changes on top.
 
-If you want to upgrade this `cilium-app` to use a newer version of cilium, you need to prepare our fork first. All changes are applied there, and then we use `vendir` to generate the contents of the `helm` folder.
+If you want to upgrade this `cilium-app` to use a newer version of cilium, you need to prepare our fork first.
+All changes are applied there, and then we use `vendir` to generate the contents of the `helm` folder in this repository.
 
-We need to create a new branch on our fork based off the tag of the version we want to upgrade to. For example, if we want to upgrade to cilium `v1.14`, we need to create a new branch based off the upstream tag for the latest `v1.14` version, which is currently `v1.14.3`.
+We need to create a new branch on our fork based off the tag of the version we want to upgrade to.
+For example, if we want to upgrade to cilium `v1.14.3`, we need to create a new branch based off the upstream tag for that version.
 ```
+git remote add upstream git@github.com:cilium/cilium.git
 git fetch upstream
 git checkout upstream/v1.14.3
 git checkout -b update-to-1-14-3
 ```
 
-Then we need to apply our custom changes on top of that new branch. Since those changes are already on the repository (although a different branch), we can use `cherry-pick` for apply those same commits to our newly created branch, for example
+Then we need to apply our custom changes on top of that new branch.
+Since those changes are already on the repository (although a different branch), we can use `cherry-pick` for apply those same commits to our newly created branch, for example
```

Comment:
I guess we can make a note here to evaluate if those custom changes are still relevant with the new release version we're trying to upgrade to